dymos/examples/double_integrator/doc/test_doc_double_integrator.py

```diff
@@ -44,8 +44,9 @@ def test_double_integrator_for_docs(self):
         # Set the options for our variables.
         #
         phase.set_time_options(fix_initial=True, fix_duration=True, units='s')
+        phase.add_state('v', fix_initial=True, fix_final=True, rate_source='u', units='m/s',
```

Comment:
is it more correct to add the shape to `u` or `v` in this case?  Adding it to `u` feels more correct, but thats just opinion.

Reply:
Yeah, that probably makes more sense.  It currently errors out if I do that, and I need to combine the fix for that problem with some of the fixes on the controls branch, so this will get taken care of on the 2nd PR.